dashToUnderscore.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+def main():
+    with open("contacts.json") as fp:
+        obj = json.load(fp)
+        underscored = dash_to_underscore(obj)
+        validate(underscored)
+        print("done")
+
+def validate(d):
+    if isinstance(d, dict):
+        for k, v in d.items():
+            if "-" in k:
+              
+                raise Exception(f"key has - in it: {k}")
+
+            validate(v)
+    
+def dash_to_underscore(obj):
+    if not isinstance(obj, dict): #stopplesing criteria
+        return obj
+
+    for k in obj.keys():
+        value = dash_to_underscore(obj[k])
+
+        if not "-" in k:
+            key = k
+        else:    
+            obj.pop(k)                        
+            key = k.replace("-","_")
+
+        obj[key]=value #recursion
+            
+    return obj
+
+    
+
+if __name__ == "__main__":
+    main()
```

Comment:
Maybe is not needed to create a separate file and include this functions in the `__init__.py`?